<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="EditorField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveFileAsAction"/>
    <addaction name="printFileAction"/>
    <addaction name="exitProgramAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="cancelAction"/>
    <addaction name="repeatAction"/>
    <addaction name="copySelectedAction"/>
    <addaction name="pasteClipboardAction"/>
    <addaction name="cutSelectedAction"/>
    <addaction name="deleteSelectedAction"/>
    <addaction name="insertDateAction"/>
   </widget>
   <widget class="QMenu" name="findMenu">
    <property name="title">
     <string>Найти</string>
    </property>
    <addaction name="findStringAction"/>
    <addaction name="gotoLineAction"/>
    <addaction name="replaceStringAction"/>
   </widget>
   <widget class="QMenu" name="infoMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="infoAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="editFontAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="menu"/>
   <addaction name="findMenu"/>
   <addaction name="infoMenu"/>
  </widget>
  <action name="newFileAction">
   <property name="text">
    <string>Новый файл</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveFileAsAction">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="printFileAction">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="exitProgramAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="cancelAction">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="copySelectedAction">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="pasteClipboardAction">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="repeatAction">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="cutSelectedAction">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="deleteSelectedAction">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="insertDateAction">
   <property name="text">
    <string>Вставить дату</string>
   </property>
  </action>
  <action name="editFontAction">
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="findStringAction">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="gotoLineAction">
   <property name="text">
    <string>Перейти к строке</string>
   </property>
  </action>
  <action name="replaceStringAction">
   <property name="text">
    <string>Заменить</string>
   </property>
  </action>
  <action name="infoAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
